rpc/Cargo.toml

```diff
@@ -18,13 +18,18 @@ trace_decoder = { workspace = true }
 serde_json = { workspace = true }
 clap = { workspace = true }
 evm_arithmetization = { workspace = true }
+mpt_trie = { workspace = true }
 thiserror = { workspace = true }
 alloy.workspace = true
 futures = { workspace = true }
 hex = "0.4.3"
 hex-literal = "0.4.1"
 itertools = "0.13.0"
 url = "2.5.0"
+primitive-types = "0.12.2"
```

Comment:
I'd prefer a `__compat_primitive_types` name, as a speed bump until we've removed it entirely